implementations/rust/ockam/ockam_core/src/lib.rs

```diff
@@ -39,6 +39,11 @@ pub extern crate hex;
 pub extern crate async_trait;
 pub use async_trait::async_trait as worker;
 
+pub extern crate ockam_macro;
+pub use ockam_macro::AsyncTryClone;
+pub extern crate futures_util;
+pub use futures_util::try_join;
```

Comment:
Can we move this `pub use` into `ockam_core::compat` for now? I'd rather not add new top level dependencies to external crates and such, and in some sense we're using this as a shim for tokio's try_join (not really, but a little).

(If that's not possible due to macro export rules, can we make it `#[doc(hidden)] pub use futures_util::try_join;`? It's mostly just an implementation artifact of the `AsyncTryClone` macro).

Reply:
Moved try_join macro to `ockam_core::compat` for now